app/build.gradle

```diff
@@ -376,4 +382,18 @@ dependencies {
     implementation("org.jetbrains.kotlin:kotlin-reflect:1.8.10")
 
     implementation "androidx.security:security-crypto:1.0.0"
+
+    //DI
```

Comment:
Please be more specific. And add space after `//` :)

Same below. A sentence or two about why we're depending on this/where we're using it would be great.

Reply:
kk, done